osschallenge/views.py

```diff
@@ -44,16 +39,40 @@ def form_valid(self, form):
     success_url = '/projects/'
 
 
+def mentor_check(group_id):
+    try:
+        mentor_id = Group.objects.get(name="Mentor").id
+    except Group.DoesNotExist:
+        return False
+    if mentor_id == group_id:
+        return True
+    return False
+
+
+def contributor_check(group_id):
+    try:
+        contributor_id = Group.objects.get(name="Contributor").id
+    except Group.DoesNotExist:
+        return False
+    if contributor_id == group_id:
+        return True
+    return False
+
+
 def ProjectIndexView(request):
     project_list = Project.objects.filter(active=True).order_by('id')
     template_name = 'osschallenge/projectindex.html'
+    try:
+        mentor = mentor_check(request.user.groups.get().id)
```

Comment:
Instead of using `mentor_check` resp. also `contributor_check` following code snippet could be used:
```python
mentor = request.user.groups.filter(name='Mentor').first()
```

Mentor is None if it doesn't exist. No exceptions need to be catched.